.github/workflows/tests.yml

```diff
@@ -68,6 +61,14 @@ jobs:
           command: composer require guzzlehttp/guzzle:^7.2 --no-interaction --no-update
         if: matrix.php >= 8
 
+      - name: Set Minimum Symfony Version
+        uses: nick-invision/retry@v1
+        with:
+          timeout_minutes: 5
+          max_attempts: 5
+          command: composer require symfony/symfony:5.4.x-dev --no-interaction --no-update
```

Comment:
why do we need this?

Reply:
PHP 8.1 will only be fully support on Symfony 5.4. This is needed for prefer-lowest builds without bumping the minimum Symfony requirement for non-PHP 8.1 installs.

Reply:
> PHP 8.1 will only be fully support on Symfony 5.4.

Oh, really? I thought Symfony treated PHP version support as a bug fix?

Reply:
Nicolas confirmed this with me.